Doc/library/concurrent.futures.rst

```diff
@@ -182,7 +182,7 @@ ThreadPoolExecutor Example
            'http://www.cnn.com/',
            'http://europe.wsj.com/',
            'http://www.bbc.co.uk/',
-           'http://some-made-up-domain.com/']
+           'http://some-made-up-domain-that-definitely-does-not-exist.com/']
```

Comment:
There is a chance that someone might claim this domain too in future. Given that the example is for illustration purpose and not a hard doctest failure I am not sure of this change.